thorlcr/slave/slavmain.cpp

```diff
@@ -1185,7 +1185,7 @@ class CKJService : public CSimpleInterfaceOf<IKJService>, implements IThreaded,
     {
         Owned<CProcessorFactory> factory = new CProcessorFactory(*this);
         processorPool.setown(createThreadPool("KJService processor pool", factory, this, keyLookupMaxProcessThreads, 10000));
-        processorPool->setStartDelayTracing(60000);
+        processorPool->setStartDelayTracing(60);
```

Comment:
whoops, good change!
Perhaps we should just default it to 60seconds in CThreadPool ?

Reply:
Sure if you want I can change default.